lib/plugins/aws/invokeLocal/index.js

```diff
@@ -74,10 +76,46 @@ class AwsInvokeLocal {
         this.options.data);
     }
 
+    if (runtime === 'python2.7') {
+      return this.invokeLocalPython(
+        handlerPath,
+        handlerName,
+        this.options.data);
+    }
+
     throw new this.serverless.classes
       .Error('You can only invoke Node.js functions locally.');
   }
 
+  invokeLocalPython(handlerPath, handlerName, event) {
+    const jsonEvent = JSON.stringify(event || {});
+
+    const context = JSON.stringify({
+      aws_request_id: 'id',
+      invokeid: 'id',
+      loggroup_name: this.provider.naming.getLogGroupName(this.options.functionObj.name),
+      log_stream_name: '2015/09/22/[HEAD]13370a84ca4ed8b77c427af260',
+      functionVersion: 'HEAD',
+      is_default_function_version: true,
+
+      function_name: this.options.functionObj.name,
+      memory_limit_in_mb: '1024',
+    });
+    const script = `
+import json
+import ${handlerPath}
+event = json.loads("""${jsonEvent}""")
+context = json.loads("""${context}""")
+context['get_remaining_time_in_millis'] = lambda: 5000
+print(${handlerPath}.${handlerName}(event, context))
+`;
```

Comment:
Thanks for jumping into this @dschep ... I know this is still WIP, but a tiny note: imo it'd be much better not to include inline python into the JS file, let's keep it in its own file like @ryansb [implemented in v0.5.6](https://github.com/serverless/serverless/blob/8239e9ca030bfbfa3e8c3f3a9d90aa6af6aec83b/bin/serverless-run-python-handler)